src/QueryPressure/Loader.cs

```diff
@@ -61,7 +61,9 @@ private ApplicationArguments Merge(string[] args)
   {
     var configExtenstions = new[] { ".yml", ".yaml" };
     var configFiles = args.Where(x => configExtenstions.Contains(Path.GetExtension(x)));
-    var scriptFile = args.Single(x => Path.GetExtension(x) == ".sql");
+
+    var scriptExtensions = new[] { ".sql", ".json" };
```

Comment:
It is an abstraction leak. Let's try to implement it using IRequirment interface

Reply:
It makes sense, I'll try to do it

Reply:
> It makes sense, I'll try to do it

you don't need to. it was decided to consider all non yml files as scripts, just pull the main branch